### PR TITLE
Implement max_time

### PR DESCRIPTION
**Pull request by mpowelson**

### CODE REVIEW

File: trajopt_sco/include/trajopt_sco/optimizers.hpp [hunk 1]
```diff
@@ -101,7 +102,8 @@ struct BasicTrustRegionSQPParameters
   double max_merit_coeff_increases;
 
   double merit_coeff_increase_ratio;  // ratio that we increate coeff each time
-  double max_time;                    // not yet implemented
+  /** @brief Max time in seconds that the optimizer will run*/
+  double max_time;
```
Comment (Levi-Armstrong) [inline trajopt_sco/include/trajopt_sco/optimizers.hpp:106]: Should this be initialized to `std::numeric_limits<double>::max()`?

**Review by Levi-Armstrong — COMMENTED** [review]
Looks good, just one comment. I did not see where the max_time was being initialized.

File: trajopt_sco/include/trajopt_sco/optimizers.hpp (hunk 1 above)
Comment (mpowelson) [reply trajopt_sco/include/trajopt_sco/optimizers.hpp:106]: Odd. Yeah it should be.

Comment (mpowelson) [reply trajopt_sco/include/trajopt_sco/optimizers.hpp:106]: Oh that is already being set in `BasicTrustRegionSQPParameters::BasicTrustRegionSQPParameters()`. That's why I didn't do it in this commit.   `max_time = static_cast<double>(INFINITY);`

Comment (Levi-Armstrong) [reply trajopt_sco/include/trajopt_sco/optimizers.hpp:106]: Ah ok.

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**